packages/solidstart/src/vite/types.ts

```diff
@@ -0,0 +1,94 @@
+export type SourceMapsOptions = {
```

Comment:
m: I very strongly recommend adding some sort of escape hatch for when people want to actually modify the underlying plugin options. We have seen that something like we do in Next.js works very well: https://github.com/getsentry/sentry-javascript/blob/3c437d18f0184b103baab76aefab0cacfa896bc7/packages/nextjs/src/config/types.ts#L330

Reply:
👍 will add